libctru/include/3ds/services/ac.h

```diff
@@ -10,11 +10,11 @@ Result acInit(void);
 /// Exits AC.
 void acExit(void);
 
+/// Waits for the system to connect to the internet.
+Result acWaitInternetConnection(void);
```

Comment:
Shouldn't this be using the uppercase notation since it's not API and just a raw IPC command ?